infoschema/perfschema/tables.go

```diff
@@ -265,7 +265,7 @@ func dataForRemoteProfile(ctx sessionctx.Context, nodeType, uri string, isGorout
 	)
 	switch nodeType {
 	case "tikv":
-		servers, err = infoschema.GetTiKVServerInfo(ctx)
+		servers, err = infoschema.GetStoreServerInfo(ctx)
```

Comment:
Is the `nodeType` also `tikv` for TiFlash?

Reply:
The `nodeType` is hard coded as `tikv` or `pd`, see: https://github.com/pingcap/tidb/blob/e2274a7790e982e4e717f6ec267af35e5f1cfedc/infoschema/perfschema/tables.go#L205-L220

Maybe we could handle it later.